### ISSUE TITLE
[fvt]2.11:There are errors when restart xcatd  for UDP service's bug

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#155]
1.download the latest build
2.on rhels 7 env:
```
[root@c910f02c01p13 ~]# cat /etc/*-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.0 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="7.0"
PRETTY_NAME="Red Hat Enterprise Linux Server 7.0 (Maipo)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.0:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.0
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION=7.0
Red Hat Enterprise Linux Server release 7.0 (Maipo)
Red Hat Enterprise Linux Server release 7.0 (Maipo)
[root@c910f02c01p13 ~]# uname -a
Linux c910f02c01p13.pok.stglabs.ibm.com 3.10.0-123.el7.ppc64 #1 SMP Mon May 5 11:18:37 EDT 2014 ppc64 ppc64 ppc64 GNU/Linux
[root@c910f02c01p13 ~]# 
```

3.Issue ``service xcatd restart``, then in ``/var/log/messages:``
```
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
Sep  7 02:41:04 c910f02c01p13 xcat[7733]: xcatd: possible BUG encountered by xCAT UDP service: Undefined subroutine &main::update_udpcontext_from_sslctl called at /opt/xcat/sbin/xcatd line 731.
```

**Comment by tingtli** [comment]
Please fix this defect in 2.9.2 for defect #191

**Comment by whowutwut** [comment]
For future, it would be good to include the ``lsxcatd -a`` output, rather than "the latest build"
